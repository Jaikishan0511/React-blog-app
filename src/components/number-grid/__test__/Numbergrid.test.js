import React from "react";
import renderer from "react-test-renderer";
// setup file
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

// test file
import { shallow, mount, render } from "enzyme";

import Numbergrid from "../index";
describe("Numbergrid", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Numbergrid />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
