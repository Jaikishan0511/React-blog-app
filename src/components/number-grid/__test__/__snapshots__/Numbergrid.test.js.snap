// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Numbergrid snapshot renders 1`] = `
<div
  className="content"
  id="content"
>
  <div
    className="row"
  >
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
    <div
      className="cell"
      id="cell"
      onClick={[Function]}
    />
  </div>
</div>
`;
